go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -601,6 +615,14 @@ func UpdateEgressFirewallRuleCount(count float64) {
 	metricEgressFirewallRuleCount.Add(count)
 }
 
+func RecordIPsecTunnelStateUpEvent(localIP, remoteIP string) {
+	metricIPsecTunnelState.WithLabelValues(localIP, remoteIP).Set(1)
```

Comment:
This won't be accepted downstream because its cardinality isnt bounded.

Reply:
yes @martinkennelly , understand your concern. the number of metrics will go out of proportion in a scaled cluster, should we generate this metric only if it's asked by the user ? or shall we report only problematic ipsec connections ?